README.md

```diff
@@ -1,699 +1,278 @@
+console.log("Unleash Byron the poodle");
 ```
 
-If the user makes an invalid selection, print out a message alerting them of the error:
-```js
-switch (snackSelection) {
-  case 'Pretzels':
-    // Vend pretzels if enough money was inserted.
-    break;
-
-  case 'Chips':
-    // Vend chips if enough money was inserted.
-    break;
-
-  case 'Water':
-    // Vend water if enough money was inserted.
-    break;
-
-  default:
-    console.error('Please select a valid snack.');
+To abstract these single actions into a collective name, we do:
 
-    break;
+```javascript
+function exerciseByronThePoodle() {
+  console.log("Wake Byron the poodle");
+  console.log("Leash Byron the poodle");
+  console.log("Walk to the park Byron the poodle");
+  console.log("Throw the fribsee for Byron the poodle");
+  console.log("Walk home with Byron the poodle");
+  console.log("Unleash Byron the poodle");
 }
 ```
 
-Inside each of those `case`s, we'll have to add some checks to make sure the user inserted enough money:
-```js
-switch (snackSelection) {
-  case 'Pretzels':
-    if (moneyInserted >= 100) {
-      // Calculate change.
+This code is a _function declaration_.
 
-      // Return snack and change to user.
-    }
+Here we have _abstracted_ 6 activities into 1 activity:
+`exerciseByronThePoodle`.
 
-    break;
+> **ASIDE**: Abstractions themselves can be lumped together _as if_ they were
+> single things.  The abstraction `dailyDogCareForByron` probably includes
+> `feedByronThePoodle`, `giveWaterToByronThePoodle` etc.
 
-  case 'Chips':
-    if (moneyInserted >= 75) {
-      // Calculate change.
-
-      // Return snack and change to user.
-    }
-
-    break;
-
-  case 'Water':
-    if (moneyInserted >= 50) {
-      // Calculate change.
-
-      // Return snack and change to user.
-    }
-
-    break;
-
-  default:
-    console.error('Please select a valid snack.');
-
-    break;
-}
-```
+## Explain How To _Call_ a Function
 
-If they inserted enough money, calculate the change, and print out a message confirming the successful transaction:
-```js
-switch (snackSelection) {
-  case 'Pretzels':
-    if (moneyInserted >= 100) {
-      const change = moneyInserted - 100;
+To execute a function you add `()` after its name. To execute the abstraction
+we just defined in JavaScript you run: `exerciseByronThePoodle()`. When we ran
+`document.querySelector()`, we were _calling_ a function. `Math.floor()` is
+another function. That `()` is also known as the _invocation operator_ because
+it tells JavaScript to...invoke the function.
 
-      console.log(`${snackSelection} dispensed. Your change is ${change}. Thank you!`);
-    }
+> **LEARNING TIP**: Feel free to define a small function in the JavaScript
+> console to test this out. You can copy the syntax provided above.
 
-    break;
+A _function_ must be _declared_ before it can be _called_. Saying
+`exerciseByronThePoodle()` before the function has been written stumps
+JavaScript.
```

Comment:
I'd say "confuses" instead of "stumps". "stumps" feels slightly colloquial and higher-level for our audience.

Reply:
✔️